parts/k8s/kubernetesmastervars.t

```diff
@@ -359,11 +359,11 @@
     "nsgID": "[resourceId('Microsoft.Network/networkSecurityGroups',variables('nsgName'))]",
 {{if not AnyAgentUsesVirtualMachineScaleSets}}
     "primaryAvailabilitySetName": "[concat('{{ (index .AgentPoolProfiles 0).Name }}-availabilitySet-',variables('nameSuffix'))]",
-    "primaryScaleSetName": "''",
+    "primaryScaleSetName": "",
```

Comment:
What is the purpose of these changes (from literal `''` to empty string)?

Reply:
If it is set to "''", it will not be an empty string in the output. the output will be ''. We need an empty string as the result.